test/common/router/BUILD

```diff
@@ -94,6 +95,15 @@ envoy_proto_library(
     ],
 )
 
+# envoy_cc_test_binary is generating mostly static binary regardless of config
```

Comment:
Is this always going to be true? Is there a way to make this more robust?

Reply:
Yes, `envoy_cc_test_binary` employs `envoy_cc_binary` which is always `linkstatic=1`.